src/main/java/com/powsybl/openloadflow/ac/outerloop/IncrementalTransformerReactivePowerControlOuterLoop.java

```diff
@@ -196,7 +194,9 @@ public OuterLoopStatus check(AcOuterLoopContext context, Reporter reporter) {
             }
         });
 
-        // Print some info
+        Reporter iterationReporter = !controlledBranchesOutOfDeadband.isEmpty() || !controlledBranchesAdjusted.isEmpty() || !controlledBranchesWithAllItsControllersToLimit.isEmpty() ?
+                Reports.createOuterLoopIterationReporter(reporter, context.getOuterLoopTotalIterations() + 1) : null;
+
         if (!controlledBranchesOutOfDeadband.isEmpty() && LOGGER.isInfoEnabled()) {
```

Comment:
Functional logging will then depends on slf4j config level which is probably not what you expect ?

Reply:
you should probably keep it but only on the largestMismatches & LOGGER.info parts

for report controlledBranchesOutOfDeadband.size() is enough in my opinion

Reply:
thank you for your suggestions, I fixed it now